shuup/admin/static_src/base/js/editor.js

```diff
@@ -64,6 +64,12 @@ function activateEditor($editor, attrs = {}) {
     $editor.parent().find(".note-editable").on("drop", (event) => {
         cancelEvent(event);
     });
+
+    $editor.parent().find(".note-codable").on("blur", function() {
+        $(this).closest(".note-editor").siblings("textarea").val(
```

Comment:
textarea can be reached like this: `const textarea = $editor.parent().find("textarea");`